### ISSUE TITLE
Add nginx for static assets

### ISSUE THREAD

**Issue by n8rzz** [openscope/openscope#166]
https://m.alphasights.com/using-nginx-on-heroku-to-serve-single-page-apps-and-avoid-cors-5d013b171a45#.fao3dls4w

https://github.com/heroku/heroku-buildpack-static

`heroku buildpacks:add https://github.com/hone/heroku-buildpack-static -a APP`

---
- [x] produce POC on `atc-dev` env
- [x] add gulp task to build/copy all assets a `/public` folder
- [x] update existing tasks to point to `/public'
- [x] add `static.json` to project
- [x] add [heroku-buildpack-static](https://github.com/heroku/heroku-buildpack-static)
    - [x] dev
    - [ ] staging
    - [ ] prod
- [x] remove Procfile
- [x] update `postinstall` npm task

---

```json
{
  "root": "public",
  "https_only": true,
  "routes": {
    "/assets/*": "/assets/",
    "/**": "index.html"
  },
  "headers": {
    "/": {
        "Cache-Control": "no-store, no-cache"
    },
    "**.js": {
        "Cache-Control": "public, max-age=512000"
    },
    "**.css": {
        "Cache-Control": "public, max-age=512000"
    },
    "/assets/**": {
      "Cache-Control": "public, max-age=31536000"
    },
    "**.json": {
        "Cache-Control": "public, max-age=31536000",
        "Content-Type": "application/json"
    },
    "**.geojson": {
        "Cache-Control": "public, max-age=31536000",
        "Content-Type": "application/json"
    }
  }
}
```

**Comment by n8rzz** [comment]
this will also give us the `https` always, re: #160

**Comment by erikquinn** [comment]
Resolved by #221.
